zproject/urls.py

```diff
@@ -588,6 +589,9 @@
 # having to rewrite URLs, and is implemented using the
 # 'override_api_url_scheme' flag passed to rest_dispatch
 urls += [
+    url(r'^user_uploads/local/([0-9A-Za-z]+)$',
+        zerver.views.upload.serve_local_file_unauthed,
+        name='zerver.views.upload.serve_local_file_unauthed'),
```

Comment:
I think probably `/local/` is the wrong work; maybe `/temporary/` would be clearer?  

This is easy to change later in any case, since things last only 60s, but I think that's more readable for what these do.

Reply:
We should probably include the original filename at the end of the URL, like the main `user_uploads` route does.

Reply:
Yeah, I agree with that; it'll help tools that try to do something with extensions.